docs/control-structures.rst

```diff
@@ -2,6 +2,56 @@
 Expressions and Control Structures
 ##################################
 
+.. index:: ! parameter, parameter;input, parameter;output
+
+Input Parameters and Output Parameters
+======================================
+
+Like in Javascript and in C, functions may take parameters as input;
+unlike in Javascript and C, they may also return arbitrary number of
+parameters as output.
+
+Input Parameters
+----------------
+
+The input parameters are declared type followed
```

Comment:
Would something like this easier to follow: "The input parameters are declared the same way as variables are. As an exception, unused parameters can omit the variable name." ?